packages/terra-switch/src/Switch.module.scss

```diff
@@ -0,0 +1,108 @@
+:local {
+  .switch-wrapper {
+    display: inline-block;
+    -webkit-tap-highlight-color: transparent;
+    width: auto;
+  }
+
+  .switch-container {
+    display: flex;
+    flex-direction: row;
+  }
+
+  .label-container {
+    display: flex;
+    flex-direction: column;
+    margin-right: 1.45rem;
+    margin-top: 0.28571rem;
+  }
+
+  .label-text {
+    color: var(--terra-switch-label-color, #1c1f21);
+    font-size: var(--terra-switch-label-font-size, 1rem);
+    font-weight: var(--terra-switch-label-font-weight, bold);
+    padding: var(--terra-switch-label-label-padding);
+    word-wrap: break-word;
+  }
+
+  .status-label-text {
+    color: var(--terra-switch-status-label-color, #1c1f21);
+    font-size: var(--terra-switch-status-label-font-size, 1rem);
+    font-weight: var(--terra-switch-status-label-font-weight, normal);
+    padding: var(--terra-switch-status-label-label-padding);
+  }
+
+  .switch {    
+    flex: 0 0 auto;
+    height: 2.92857rem;
+    margin-top: 0.28571rem;
+    outline: none;
+    position: relative;
+    width: 6.14285rem;
+  }
+
+  .selected {
+    background-color: #fff;
+  }
+
+  .unselected {
+    background-color: var(--terra-switch-tray-background-color, rgba(74, 74, 74, 0.2));
+  }
+
+  [aria-checked='true'] {
+    transform: translateX(3.17142rem);
+  }
```

Comment:
This might have some strange global effects without an associated class name.

Reply:
Added Class [here](https://github.com/cerner/terra-core/blob/3ccbde10c12629e3a8a6a617536b9c403ec693c7/packages/terra-switch/src/Switch.module.scss#L52)